### PR TITLE
Adds button invert support and auto ANALOG_MAX assignment

### PR DESCRIPTION
**Pull request by LittleNyanCat**
Pretty much title. 

At least one person right now need that button invert and it might be useful for someone else eventually

Setting ANALOG_MAX automatically is one less thing for users to mess up, and can be overrided if needed

### CODE REVIEW

File: firmware/lucidgloves-firmware/AdvancedConfig.h
Comment (danwillm) [inline firmware/lucidgloves-firmware/AdvancedConfig.h:36]: If we're defining the `ANALOG_MAX` in `AdvancedConfig.h` rather than the main config file, it might be useful to mention where to set `ANALOG_MAX` in the error message.

**Review by danwillm — CHANGES_REQUESTED** [review]
Other changes lgtm, but will need to wait for Lucas' review before merging.

File: firmware/lucidgloves-firmware/AdvancedConfig.h
Comment (LittleNyanCat) [reply firmware/lucidgloves-firmware/AdvancedConfig.h:36]: Good catch, I originally had the error say "on the line above" but I ended up getting rid of that

**Review by lucas-vrtech — APPROVED** [review]
Looks good!

**Pull request merged**
